### PR TITLE
Rover: Notify lat and long to the seventh decimal place

### PR DESCRIPTION
**Pull request by muramura**
The seventh decimal place corresponds to centimeters.

### CODE REVIEW

**Review by khancyr — APPROVED** [review]
Fit in a single message anyway. LGTM

**Review by peterbarker — APPROVED** [review]

**Comment by rmackay9** [comment]
Does it actually display all the digits?  I suspect it does in which case I should do the same in the recent slung payload simulator.

**Comment by peterbarker** [comment]
> Does it actually display all the digits? I suspect it does in which case I should do the same in the recent slung payload simulator.

I haven't provoked it to do so, but we may as well give it every chance.

File: Rover/commands.cpp [hunk 1]
```diff
@@ -33,7 +33,7 @@ bool Rover::set_home(const Location& loc, bool lock)
     mode_auto.mission.write_home_to_storage();
 
     // send text of home position to ground stations
-    gcs().send_text(MAV_SEVERITY_INFO, "Set HOME to %.6f %.6f at %.2fm",
+    gcs().send_text(MAV_SEVERITY_INFO, "Set HOME to %.7f %.7f at %.2fm",
```
Comment (tridge) [inline Rover/commands.cpp:36]: this won't work on ChibiOS at least, possibly not on SITL too, we go via a float inside the printf engine

**Comment by rmackay9** [comment]
We think this doesn't actually work.  Remember to provide testing results!

We discussed on the dev call that what we need is a function somewhere that allows printing lat,lon values to the required number of decimal places

**Pull request closed without merging**